plugins/interactive.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: interactive
+spec:
+  version: v1.0.3
+  homepage: https://github.com/similarweb/kubectl-interactive
+  shortDescription: A kubectl plugin that provides interactive, easy-to-use and flexible interface
+  description: |
+    A kubectl plugin that is easy-to-use, flexible, and interactive
+    You will have the ability to roam around the Kubernetes world with a blazing fast selection menu
```

Comment:
again the description doesn't explain anything specific and it has multiple subjective statements such as _easy-to-use, flexible, and blazing fast_.

please explain what the command actually does.

Reply:
changed, hope it makes sense now